### ISSUE TITLE
Error when using color picker

### ISSUE THREAD

**Issue by WebFreak001** [ShareX/ShareX#788]
```
System.ArgumentOutOfRangeException: Parameter must be positive and < Width.
Parameter name: x
   at System.Drawing.Bitmap.GetPixel(Int32 x, Int32 y)
   at ShareX.ScreenCaptureLib.RectangleRegion.get_CurrentColor()
   at ShareX.TaskHelpers.SelectPointColor()
   at ShareX.ScreenColorPicker.btnPipette_Click(Object sender, EventArgs e)
   at System.Windows.Forms.Button.OnMouseUp(MouseEventArgs mevent)
   at System.Windows.Forms.Control.WmMouseUp(Message& m, MouseButtons button, Int32 clicks)
   at System.Windows.Forms.Control.WndProc(Message& m)
   at System.Windows.Forms.ButtonBase.WndProc(Message& m)
   at System.Windows.Forms.Button.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)
```

This happens when using the button right to Start color picking in the Color picker menu. It only happens when picking a color on my secondary monitor. Not reproducable in windows 8, but reproducable on my friends windows 7 computer

**Comment by Scrxtchy** [comment]
Duplication of #700

**Comment by Jaex** [comment]
Your friend must be using old version.

**Comment by WebFreak001** [comment]
v10.0

**Comment by Jaex** [comment]
Why you closed this then?

**Comment by WebFreak001** [comment]
because #700 was the same and it was resolved

**Comment by Jaex** [comment]
That fix was in v10.0 if it still not works then it is different.
